<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>251</width>
         <height>431</height>
        </rect>
       </property>
       <property name="title">
        <string>随机数</string>
       </property>
       <widget class="QListWidget" name="rand_list">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>231</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>266</x>
         <y>9</y>
         <width>251</width>
         <height>431</height>
        </rect>
       </property>
       <property name="title">
        <string>冒泡排序结果</string>
       </property>
       <widget class="QListWidget" name="bubble_list">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>231</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>522</x>
         <y>9</y>
         <width>251</width>
         <height>431</height>
        </rect>
       </property>
       <property name="title">
        <string>快速排序结果</string>
       </property>
       <widget class="QListWidget" name="quick_list">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>231</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="start">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>251</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
